Changelog.md

```diff
@@ -14,11 +14,15 @@
 
 ### Bug Fixes
 
+- [#755]: Fix incorrect missing of trimming all-space text events when
+  `trim_text_start = false` and `trim_text_end = true`.
+
```

Comment:
Re: " events should not be trimmed at boundary of text / CDATA, or text / PI, or text / comment in some cases", is there an issue filed for that with some examples of such boundary cases?

Reply:
No, such issue does not exist. I made investigation in #520 for serde deserializer and implement reading text properly. I cross-checked that using Java XmlBeans library.